### PR TITLE
Gives service borgs a chisel

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request
See title
Does not include a way for them to make blocks.
## Why It's Good For The Game
Service borgs making art is fun and thematic 
Relies on a carbon to make blocks for you so you can't waste the whole ORM making statues. 
## Changelog
:cl:
add: Service borgs have a chisel now, make some statues!
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot_model.dm [hunk 1]
```diff
@@ -744,7 +744,8 @@
 		/obj/item/reagent_containers/borghypo/borgshaker,
 		/obj/item/borg/lollipop,
 		/obj/item/stack/pipe_cleaner_coil/cyborg,
-		/obj/item/borg/apparatus/beaker/service)
+		/obj/item/borg/apparatus/beaker/service,
+		/obj/item/chisel)
```
Comment (dragomagol) [inline code/modules/mob/living/silicon/robot/robot_model.dm:748]:
```suggestion
		/obj/item/chisel,
	)
```

**Review by dragomagol — APPROVED** [review]

**Comment by RaveRadbury** [comment]
Detroit Become Human RP intensifies.

**Comment by KittyNoodle** [comment]
smh ai art in ss13 now

**Comment by Iamgoofball** [comment]
this is a violation of blade runner lore

**Comment by BlueMemesauce** [comment]
I sure hope people don't have an entire argument calling this cosmetic feature OP and force it to be a upgrade again like with #53183

**Comment by ATH1909** [comment]
do they have a way to anchor chiseling blocks?

... do chiseling blocks start anchored? it's been a while since I've used them.

**Comment by YakumoChen** [comment]
can just get another borg to help wrench it, or a human

you could get an engi borg to make you blocks too, I suppose.

**Comment by theselfish** [comment]
> Detroit Become Human RP intensifies.

image

**Pull request merged**